db/local_db/status.go

```diff
@@ -36,3 +42,41 @@ func GetStatus() (*RunningDBInstanceInfo, error) {
 
 	return info, nil
 }
+
+// errorIfUnknownService errors if it can find a `postmaster.pid` in the `INSTALL_DIR`
+// and the PID recorded in the found `postmaster.pid` is running
```

Comment:
why does it error?

unclear from comment the reasoning here

Reply:
updated comment